task/buildah-10gb/0.1/patch.yaml

```diff
@@ -7,3 +7,9 @@
 - op: replace
   path: /spec/steps/0/computeResources/requests/memory
   value: 8Gi
+- op: add
+  path: /metadata/annotations/build.appstudio.redhat.com~1expires-on
+  value: "2025-01-21T00:00:00Z"
+- op: add
+  path: /metadata/annotations/build.appstudio.redhat.com~1expiry-message
+  value: "Instead of using this task, switch to buildah task and configure build step resources as described in the doc: https://konflux-ci.dev/docs/how-tos/configuring/overriding-compute-resources/"
```

Comment:
optional: we could also link the quay repo for the buildah task to help users find the right bundle ref https://quay.io/repository/konflux-ci/tekton-catalog/task-buildah

Reply:
Makes sense.